app/models/piece.rb

```diff
@@ -68,11 +68,24 @@ def diagonal?(new_x, new_y, old_x, old_y)
     absolute_x == absolute_y ? true : false
   end
 
-  def piece_here?(x, y)
-    if Piece.where("x_coordinate = ? AND y_coordinate = ?", x, y) == []
+  def piece_here?(x, y, game)
+    if Piece.where(game_id: game_id, x_coordinate: x, y_coordinate: y) == []
       return false
     else
       return true
     end
   end
+
+
+  def move_to(x, y)
+    if piece_here?(x, y, self.game_id)
```

Comment:
I am wondering if we could omit the game in the parameter.. `piece_here?` can just use `self.game_id` to figure out the game...